recipes/argon2/all/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.1.2)
```

Comment:
~~Cosmetic but since there are other changes to apply....~~

```suggestion
cmake_minimum_required(VERSION 3.1)
```

**this file can be deleted since it's not used**